polkadot/xcm/docs/src/guides/mod.rs

```diff
@@ -0,0 +1,5 @@
+//! # XCM Guides
+//!
+//! These guides aim to get you up and running with XCM.
+//!
+//! Coming soon.
```

Comment:
what topics do you intend to cover here?

Reply:
Would be great to collect some asset guide ideas here https://github.com/paritytech/polkadot-sdk/issues/4377